### ISSUE TITLE
This line can be shorter :)

### ISSUE THREAD

**Issue by LordLukin** [jzych/KursCPP-UniversityDB2#19]
https://github.com/jzych/KursCPP-UniversityDB2/blob/71fd945a4ce033dceebb4e2ebead870355b1191e/PeselValidation.cpp#L63

What about `return gender == pesel_gender`?

**Comment by jzych** [comment]
True, but ?: operator may be unknown to most of my colleagues so full development makes it easier to read and understand.

**Comment by LordLukin** [comment]
For me not using unnecesary code is better ;)

**Comment by jzych** [comment]
I got below error when tried implement shorter version:
> error: ISO C++ does not allow ?: with omitted middle operand [-Werror=pedantic]

**Comment by LordLukin** [comment]
I guess you did something wrong :)
`return gender == pesel_gender` should be completely legal. Unless you put question mark somewhere? ;)
